node/cli/src/command.rs

```diff
@@ -137,6 +138,11 @@ impl SubstrateCli for Cli {
 	}
 
 	fn load_spec(&self, id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, String> {
+		if let Some(subcommand) = &self.subcommand {
+			if let Subcommand::PerfTest(_) = subcommand {
+				return Ok(Box::new(chain_spec::moonbase::development_chain_spec(None, None)));
+			}
+		}
```

Comment:
```suggestion
		if let Some(Subcommand::PerfTest(_)) = &self.subcommand {
				return Ok(Box::new(chain_spec::moonbase::development_chain_spec(None, None)));
		}
```
Just a quick comment about the pattern matching. I haven't yet understood why this change is necessary.

Reply:
I need to revisit this. The basic problem I'm trying to solve is that the chain should implicitly be moonbase-dev in this case and `load_spec` really doesn't like having no chain specified, e.g.:

```rust
      if id.is_empty() {
          return Err("Not specific which chain to run.".into());
      }
```

Maybe we do want to allow different chains, though...